recipes/cdat-lite/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "6.0rc2" %}
+
+package:
+    name: cdat-lite
+    version: {{ version }}
+
+source:
+    fn: cdat_lite-{{ version }}.tar.gz
+    url: http://ndg.nerc.ac.uk/dist/cdat_lite-{{ version }}.tar.gz
+    sha256: 6cbfdf9b47fdbc189ca7d2819dafb6cf958116e0f6077b325c375a0ddfab2a95
+
+build:
+    number: 0
+    skip: True  # [win or py3k]
+    script:
+        - export NETCDF_HOME=$PREFIX
+        - export HDF5_HOME=$PREFIX
+        - python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - numpy 1.8.*  # The oldnumeric module was removed in numpy 1.9.
```

Comment:
Oh boy, this is going to be special. Maybe we will want to build a copy of this ourselves too.